pkg/clusteragent/clusterchecks/status_templates/clusterchecks.tmpl

```diff
@@ -0,0 +1,23 @@
+{{- if .clusterchecks }}
```

Comment:
This template and the rest have been copied from the existing ones.
I see that the original ones have not been deleted. Are they still needed? If both are needed it'd be good to specify somewhere that we'll need to update both every time that we make a change.

Reply:
Since I'm migrating one status command at a time, I need to keep the other templates for now. So, the existing templates at `pkg/status/render/templates/*` can not be deleted yet, as I want the other status command to work in isolation from the agent status command. I will remove those once I migrate all status commands.

Reply:
I will update the PR description to include this information 😄